packages/components/src/post-stats-card/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
+import { Card } from '../';
+import './style.scss';
+
+type PostStatsCardProps = {
+	commentCount: number | null;
+	heading: React.ReactNode;
+	likeCount: number | null;
+	viewCount: number | null;
+	post: {
+		date: string;
+		post_thumbnail: string | null;
+		title: string;
+	};
+};
+
+export default function PostStatsCard( {
+	commentCount,
+	heading,
+	likeCount,
+	post,
+	viewCount,
+}: PostStatsCardProps ) {
+	const translate = useTranslate();
+	const parsedDate = useMemo( () => new Date( post?.date ).toLocaleDateString(), [ post?.date ] );
+	return (
+		<Card className="post-stats-card">
+			<div className="post-stats-card__heading">{ heading }</div>
+			<div className="post-stats-card__post-info">
+				<div className="post-stats-card__post-title">{ post?.title }</div>
+				{ post?.date && (
+					<div className="post-stats-card__post-date">
+						{ translate( 'Published %(date)s', {
+							// TODO: Show relative duration instead of published date
+							args: { date: parsedDate },
```

Comment:
Possible enhancement would be to put the real date value in a tooltip.

Reply:
Great idea, updated the todo comment.